voyage-model-provider/main.py

```diff
@@ -0,0 +1,72 @@
+import json
+import os
+
+from voyageai import AsyncClient
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import JSONResponse
+
+debug = os.environ.get("GPTSCRIPT_DEBUG", "false") == "true"
+client = AsyncClient(api_key=os.environ.get("OTTO8_VOYAGE_MODEL_PROVIDER_API_KEY", "pa-P3XFfrvvOj1hzJSdUmkV_O7RVngUQ-00HpsDzgRX8sc"))
```

Comment:
There’s a hardcoded key here. Was this intentional?

Reply:
Good lord. What was I doing. No.